packages/select/src/types.ts

```diff
@@ -66,15 +76,34 @@ export type RematchSelect<M extends object = Models, RootState = any> = ((
 	StoreSelectors<RootState>
 
 declare module '@rematch/core' {
-	interface Model<
+	export interface Model<
```

Comment:
I think we don't need to complete the props of Model, these props can be combined when Model definition is used

Reply:
could you give me an example? i'm not sure how to fill 

`selectors?: ModelSelectorsConfig<TState>`, for the model